### PR TITLE
Add repair size information to health checker

### PR DESCRIPTION
**Pull request by MSevey**
Adding the repair remaining size to the output with the number of siafiles with poor health

### CODE REVIEW

File: setup-scripts/health-checker.py [hunk 1]
```diff
@@ -277,6 +282,23 @@ async def check_alerts():
         if contains_string(line, health_of):
             siafile_alerts.append(line)
 
+    # Repair Size 
+    # Execute 'siac renter' and read the response
+    cmd_string = "docker exec {} siac renter".format(CONTAINER_NAME)
+    siac_alert_output = os.popen(cmd_string).read().strip()
```
Comment (m-cat) [inline setup-scripts/health-checker.py:288]:
```suggestion
    siac_renter_output = os.popen(cmd_string).read().strip()
```

**Review by m-cat — DISMISSED** [review]
One comment, can be a follow-up if this is urgent

**Comment by MSevey** [comment]
> One comment, can be a follow-up if this is urgent

Addressed

File: setup-scripts/health-checker.py [hunk 2]
```diff
@@ -277,6 +282,23 @@ async def check_alerts():
...
+    siac_renter_output = os.popen(cmd_string).read().strip()
+
+    # Initialize variables
+    repair_remaining = ''
+
+    # Pattern strings to search for
+    repair_str = 'Repair Data Remaining'
+    
+    # Split the output by line and check for the repair remaining
+    for line in siac_alert_output.split("\n"):
```
Comment (MSevey) [inline setup-scripts/health-checker.py:297]:
```suggestion
    for line in siac_renter_output.split("\n"):
```

**Review by m-cat — APPROVED** [review]

**Review by ChrisSchinnerl — APPROVED** [review]

**Pull request merged**